src/radio/utils/PlaylistParser.js

```diff
@@ -0,0 +1,52 @@
+/* @flow */
+
+const https = require('https');
+const querystring = require('querystring');
+
+export default function parsePlaylist(
+  id: string,
+  token: string = '',
```

Comment:
i don't think token needs a default
could just make it `token?` (this is a flow thing not a js thing)